packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,7 +146,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     return (
       <div ref='root' className={ css('ms-Dropdown-container') }>
         { label && (
-          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
+          <Label className={ css('ms-Dropdown-label') } id={ id + '-label' } htmlFor={ id } ref={ this._resolveRef('_dropdownLabel') } required={ required }>{ label }</Label>
```

Comment:
I tested using narrator on the dev-deploy link, the label is read out as expected.

But the first person in the list is focused by default and that person's information is still not read not, I suppose it should be added to this label. @ipip2005 , do you have other considerations?